src/pluto/exception.cr

```diff
@@ -1,5 +1,9 @@
 class Pluto::Exception < ::Exception
-  getter error_code : Int32
+  getter error_code : Int32?
+
+  def initialize(message : String | Nil = nil, cause : Exception | Nil = nil)
+    super(message, cause)
```

Comment:
We probably want to ensure at least `message` or `cause` are non-nil for the sake of providing useful information to the developer on why the exception was thrown.